src/lib/SocialIcon.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts">
+  import Fa from 'svelte-fa/src/fa.svelte'
+  import { iconMapping } from './iconMapping'
+
+  export let social: keyof typeof iconMapping = 'Website'
+  export let id: string | undefined = undefined
+  export let link: string | undefined = undefined
+  let titleOverwrite: string | undefined = undefined
+  export let noLink = false
+  let finalLink = link
+
+  $: {
+    if (!noLink && !link && id) {
+      switch (social) {
+        case 'Discord':
+          finalLink = `https://discord.gg/${id}`
```

Comment:
I think this is the link you want:

```ts
finalLink = `https://discord.com/users/${id}` 
```

What you have now gets interpreted as an invite.

Reply:
True, fixed!